test/avocado/run-tests

```diff
@@ -43,12 +43,15 @@ if not arg_attachments_dir:
 if not os.path.exists(arg_attachments_dir):
     os.makedirs(arg_attachments_dir)
 
+
 def prepare_kdump_test(machine):
     # enable kernel kdump support
+    machine.execute("rpm -q nfs-utils || yum install -y nfs-utils", timeout=500)
```

Comment:
Note to self: nfs-utils is pre-installed on our fedora-30 image, so this is ok.

Reply:
okay, I can remove this. I know that this should be satisfied almost everywhrere, but I want to ensute this

Reply:
No, please don't remove it, it's useful for  downstream testing. This was really just a note to myself that this won't cause internet access on our CI.